### ISSUE TITLE
How to use clip on chinese dataset?

### ISSUE THREAD

**Issue by zhouwei5113** [Zasder3/train-CLIP#28]
How to use clip on chinese dataset? Should I change txt_encoder pretrain model with a chinese version?

**Comment by Zasder3** [comment]
Exactly! I think that's the only necessary change. Let me know how it goes :)

**Comment by zhouwei5113** [comment]
I found a default learning rate 3e-3 when using train_finetune.py, which is a suggested learning rate for both image and text encoder, right? @Zasder3

**Comment by zhouwei5113** [comment]
> Exactly! I think that's the only necessary change. Let me know how it goes :)

Training on chinese dataset is very difficult to converge...

**Comment by Zasder3** [comment]
Bit late to this! An lr that I use frequently is `1e-4`, that or something in that family typically gives good results.

Hopefully future users will be able to benefit from your experiments.

**Comment by yangapku** [comment]
@zhouwei5113 @Zasder3 Hi, maybe you can refer to this repo! https://github.com/OFA-Sys/Chinese-CLIP
